notes.md

```diff
@@ -159,81 +160,81 @@ The StereoMode used to be part of Matroska v2 but it didn't meet the requirement
 
 ## Block Timecodes
 
-The Block's timecode is signed integer that represents the Raw Timecode relative to the Cluster's Timecode, multiplied by the TimecodeScale (see the TimecodeScale notes.
+The `Block Element`'s timecode MUST be a signed integer that represents the `Raw Timecode` relative to the `Cluster`'s `Timecode Element`, multiplied by the `TimecodeScale Element`. See [TimecodeScale](#timecodescale) for more information.
 
-The Block's timecode is represented by a 16bit signed integer (sint16). This means that the Block's timecode has a range of -32768 to +32767 units. When using the default value of TimecodeScale, each integer represents 1ms. So, the maximum time span of Blocks in a Cluster using the default TimecodeScale of 1ms is 65536ms.
+The `Block Element`'s timecode MUST be represented by a 16bit signed integer (sint16). The `Block`'s timecode has a range of -32768 to +32767 units. When using the default value of the `TimecodeScale Element`, each integer represents 1ms. The maximum time span of `Block Elements` in a `Cluster` using the default `TimecodeScale Element` of 1ms is 65536ms.
 
-If a Cluster's Timecode is set to zero, it is possible to have Blocks with a negative Raw Timecode. Blocks with a negative Raw Timecode are not valid.
+If a `Cluster`'s `Timecode Element` is set to zero, it is possible to have `Block Elements` with a negative `Raw Timecode`. `Block Elements` with a negative `Raw Timecode` are not valid.
 
 ## Raw Timecode
 
-The exact time of an object represented in nanoseconds. To find out a Block's Raw Timecode, you need the Block's timecode, the Cluster's Timecode, and the TimecodeScale. For calculation, please see the see the TimecodeScale notes.
+The exact time of an object SHOULD be represented in nanoseconds. To find out a `Block`'s `Raw Timecode`, you need the `Block`'s `Timecode Element`, the `Cluster`'s `Timecode Element`, and the `TimecodeScale Element`. For calculation, please see the see the [`TimecodeScale` notes](#timecodescale).
```

Comment:
>For calculation, please see

This sounds strange to me, because it’s actually the following ¶

Reply:
Just gonna drop the "see the TimecodeScale notes" sentence